### PR TITLE
Add RHEL and Windows x86 to CI.

### PR DESCRIPTION
**Pull request by Sridhar-MS**
- Changes in netci.groovy to delete post-merge builds. We are using VSO
  for that.
- Also add RHEL and Windows x86 to PR CI jobs. Instead of doing 'debug'
  and 'release' for all platforms we selectively have a mixture
  of debug and release builds. (For ubuntu we are doing debug and release
  but that will change when we have 'Debian 8.2' support. Ubuntu:Debug
  will be replaced by Debian8.2:Debug)
- Minor change in install.sh to maintain uniformity in OS names.

cc: @mmitche for netci.groovy changes
cc: @anurse @eerhardt @brthor @piotrpMSFT

### CODE REVIEW

**Comment by Sridhar-MS** [comment]
@dotnet-bot test ci please

**Comment by brthor** [comment]
:shipit:

**Comment by eerhardt** [comment]
:+1:

**Comment by piotrpMSFT** [comment]
@Sridhar-MS can you try rebasing atop my latest commit? it should fix the OS X issue.

**Comment by Sridhar-MS** [comment]
@dotnet-bot test OSX Debug Build please

**Comment by Sridhar-MS** [comment]
@dotnet-bot test OSX Release Build please

**Comment by Sridhar-MS** [comment]
@dotnet-bot test ci please

**Comment by Sridhar-MS** [comment]
@mmitche Can I get a :shipit: for the groovy file?

File: netci.groovy [hunk 1]
```diff
@@ -7,61 +7,56 @@ import jobs.generation.Utilities;
 
 def project = GithubProject
 def branch = GithubBranchName
+def isPR = true
```
Comment (mmitche) [inline netci.groovy]: What is this for?

Comment (Sridhar-MS) [inline netci.groovy]: Just to make explicit that this groovy file generates CI projects only for PRs.

**Comment by mmitche** [comment]
Minor nit, otherwise LGTM

File: netci.groovy (hunk 1 above)
Comment (mmitche) [inline netci.groovy]: Ahh okay.

**Pull request merged**